dbt/scripts/run_iasworld_data_tests.py

```diff
@@ -1228,6 +1325,19 @@ def get_test_categories(
         tablename = get_tablename_from_node(node)
         column_name = get_column_name_from_node(node)
         test_description = meta.get("description")
+        notify = meta.get("notify")
+        if notify:
+            # We can't use variables directly in dbt meta configs, because the
+            # `var()` function doesn't get called during compilation there.
+            # Instead, interpret the `notify` attribute as a string representing
+            # the name of the variable storing the SNS topic that we need
+            # to publish to
+            notify_var = get_var_value_from_run_results(notify, run_results)
```

Comment:
This limitation is a huge bummer. Parsing the variables by name (rather than relying on `dbt compile` to build them into the DAG) felt like the best workaround to me, but I'm open to other ideas.